### DISCUSSION TITLE
synced folder structure

### DISCUSSION THREAD

**Discussion by acherley** [remotely-save/remotely-save#94] [Q&A] [answered]
this plugin is very useful, I used the s3 to do the sync.
I have a question, should I create a s3 bucket for one vault ?
does use same s3 bucket to sync all vaults ? such as create and save to the folder with vault name

**Comment by fyears** [comment, marked as answer]
Currently one bucket for one vault, sorry.

In the future, I may support one bucket for multiple vaults. But no accurate plan yet.

**Reply by acherley** [reply]
Thanks.

**Comment by fyears** [comment]
new version supports this
